spec/spec_helper.rb

```diff
@@ -1,5 +1,10 @@
 # frozen_string_literal: true
 
+if ENV.fetch('COVERAGE', nil) == 'true'
```

Comment:
It would be best to leave a comment as shown here:
https://github.com/rubocop/rubocop/blob/d0bc9e161c936e104e12a90ea09382578ce4964a/Rakefile#L5

Reply:
Done.